packages/common/container-definitions/src/runtime.ts

```diff
@@ -161,14 +160,6 @@ export interface IContainerContext {
 	readonly closeFn: (error?: ICriticalContainerError) => void;
 	readonly deltaManager: IDeltaManager<ISequencedDocumentMessage, IDocumentMessage>;
 	readonly quorum: IQuorumClients;
-	/**
-	 * @deprecated This method is provided as a migration tool for customers currently reading the code details
```

Comment:
For the stragglers that are still using this shim, what will happen once it's gone?

Reply:
We already have a preferred solution in place for known usage (implemented at the time of the deprecation) - this API was left in place as a fallback while deployments got up to date, but telemetry shows negligible remaining usage now (and even in those cases, has reasonable fallback).